qemu/tests/cfg/blockdev_commit_query_named_block_nodes.cfg

```diff
@@ -0,0 +1,38 @@
+- blockdev_commit_query_named_block_nodes:
+    type = blockdev_commit_query_named_block_nodes
+    virt_test_type = qemu
+    only Linux
+    images += " data"
+    force_create_image_data = yes
+    force_remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags = sn1 sn2 sn3 sn4
+
+    image_size_sn1 = 500M
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    image_name_sn2 = sn2
+    image_size_sn2 = 500M
+    image_format_sn2 = qcow2
+
+    image_name_sn3 = sn3
+    image_size_sn3 = 500M
+    image_format_sn3 = qcow2
+
+    image_name_sn4 = sn4
+    image_size_sn4 = 500M
+    image_format_sn4 = qcow2
+
+    device_tag = "data"
+    format = qcow2
+    rebase_mode = unsafe
+    query_named_block_nodes = yes
```

Comment:
This tc is designed for query blocks when committing, so remove query_named_block_nodes?

Reply:
Have update.